### ISSUE TITLE
Doubts related to code. 

### ISSUE THREAD

**Issue by ashimagarg** [kevinzakka/style-transfer#4]
Hello, 
1. You have used self.output_img = K.placeholder((1, new_dims[0], new_dims[1], 3)) to define combined image. And x = np.random.uniform(0, 255, (1, self.img_nrows, self.img_ncols, 3)) - 128 to feed values to placeholder. Shouldn't output_img be defined as K.Variable as it needs to be trained to get the final output?? 

2. Also, I have a problem: how to modify the code to use keras.optimizers.Adam instead of calculating gradients and then optimizing using fmin_l_bfgs_b from scipy.optimize .
